agent/php_execute.h

```diff
@@ -24,6 +24,8 @@
 #define OP_ARRAY_IS_METHOD(OP, FNAME) \
   (0 == nr_strcmp(nr_php_op_array_function_name(OP), (FNAME)))
 
+#define CLM_STRLEN_MAX (255)
+
```

Comment:
Maybe this could be moved closer to where it's needed? Maybe [here](https://github.com/zsistla/newrelic-php-agent/blob/clm_to_dev/agent/php_execute.c#L1098)? It's not needed anywhere else in the codebase.

Reply:
Please see: [6c23e7a](https://github.com/newrelic/newrelic-php-agent/pull/585/commits/6c23e7a214ef06257731f1b3c1e3bb03d0be49cd)